protos/sources/vega/vega.proto

```diff
@@ -1063,6 +1065,8 @@ message LiquidityProviderFeeShare {
   string average_entry_valuation = 3;
   // Average liquidity score.
   string average_score = 4;
+  // The virtual stake of this liquidity provider (float).
```

Comment:
What does (float) mean here?

Reply:
string representing a decimal. I included it here (and in the graphQL schema) because one of the other fields had that bit included.